network-api/networkapi/minisites/templates/minisites/blocks/image_text_block.html

```diff
@@ -0,0 +1,8 @@
+{% load wagtailcore_tags wagtailimages_tags %}
+
+<div>
+	{% block prefix %}{% endblock %}
```

Comment:
We typically use 2 space indentation...should we add linting for that?

Reply:
It's not caught by `flake8` at the moment, so we might be able to do that if we can run an indent linting on `*.html` inside the `/network-api/networkapi/` dir. I guess the question would be what we're linting for: as code "on the python side" tabs are pretty PEP8-conformant, so it might actually be more worth it to see if we can make sure that final page HTML as generated by wagtail is nicely indented without changing the markup semantics?